### PR TITLE
hack: add HACK_MINIMIZE command to make running on libvirt possible.

### PR DESCRIPTION
**Pull request by squeed**
This is an awful hack, but it is fast(er).

Also, wait for the network operator namespace to come in to existence before continuing.

### CODE REVIEW

**Comment by squeed** [comment]
I'm not particularly proud of this.

File: hack/overrides-minimize-patch.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# When doing rapid local development on libvirt, it's nice to remove
+# some of the more resource-heavy components.
+# To use this, set the environment variable HACK_MINIMIZE
```
Comment (danwinship) [inline hack/overrides-minimize-patch.yaml]: document in INSTALLER-HACKING.md?

Comment (squeed) [reply hack/overrides-minimize-patch.yaml]: I was a bit embarrassed, but done.

**Pull request merged**